vaccine_feed_ingest/runners/nc/myspot_gov/normalize.py

```diff
@@ -43,27 +45,42 @@ def _get_location(site: dict):
 def _get_contacts(site: dict):
     ret = []
     if site["Appointment Phone"] != "":
-        raw_phone = str(site["Appointment Phone"]).lstrip("1")
-        if raw_phone[3] == "-" or raw_phone[7] == "-":
-            phone = "(" + raw_phone[0:3] + ") " + raw_phone[4:7] + "-" + raw_phone[8:12]
-            phone_notes = raw_phone[12:]
-        elif len(raw_phone) == 10:
-            phone = "(" + raw_phone[0:3] + ") " + raw_phone[3:6] + "-" + raw_phone[6:10]
-            phone_notes = ""
+        ret.extend(_parse_phone_numbers(site["Appointment Phone"]))
+
+    url = site["Web Address"]
+    # Some URLs have multiple schemes.
+    valid_url = re.match(r"(https?:\/\[/)*:(.+)", url)
```

Comment:
Why is there a `\[` in the regex? Should this be more like:
```suggestion
    valid_url = re.match(r"(https?:\/\/)?(.+)", url)
```

Reply:
That's bizarre and I'm glad that you caught it before we threw out a bunch of websites. I'm going to verify that this still works.

Reply:
And indeed, it stopped working!